spec/unit/cloudkeeper/aws/cloud_spec.rb

```diff
@@ -109,86 +109,6 @@ module Aws
           end
         end
       end
-
```

Comment:
Why did you remove these tests? Sorry, I know we've talked about it but I can't remember.